4.0.8 fails to compile with "Syntax error: end of file unexpected"

Running `cargo build` on a project that depends on `servo-fontconfig-sys` results in the following error:

```
error: failed to run custom build command for `servo-fontconfig-sys v4.0.8`

Caused by:
  process didn't exit successfully: `/home/anon/Projects/amethyst/target/debug/build/servo-fontconfig-sys-7e9ac9023a489739/build-script-build` (exit code: 101)
--- stdout
cd /home/anon/Projects/amethyst/target/debug/build/servo-fontconfig-sys-bc54acb2bed42777/out
&& \
        CC="gcc" \
        AR="ar" \
        FREETYPE_CFLAGS="" \
        FREETYPE_LIBS="" \
        CFLAGS=""" -fPIC" \
        /home/anon/.cargo/registry/src/github.com-1ecc6299db9ec823/servo-fontconfig-sys-4.0.8/configure \
        --disable-docs \
        --disable-shared \
         \
        --host=x86_64-unknown-linux-gnu --sysconfdir=/etc --localstatedir=/var &&

--- stderr
/bin/sh: 11: Syntax error: end of file unexpected
make: *** [makefile.cargo:83: /home/anon/Projects/amethyst/target/debug/build/servo-fontconfig-sys-bc54acb2bed42777/out/Makefile] Error 2
thread 'main' panicked at 'assertion failed: Command::new("make").env("MAKEFLAGS",
                         env::var("CARGO_MAKEFLAGS").unwrap_or_default()).args(&["-R",
                                                                                 "-f",
                                                                                 "makefile.cargo"]).status().unwrap().success()', /home/anon/.cargo/registry/src/github.com-1ecc6299db9ec823/servo-fontconfig-sys-4.0.8/build.rs:20:5
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace.

warning: build failed, waiting for other jobs to finish...
error: build failed
```

This does not happen if I downgrade to 4.0.7 with `cargo update -p servo-fontconfig-sys --precise 4.0.7`.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "servo-fontconfig-sys"
-version = "4.0.8"
+version = "4.0.9"
 authors = ["Keith Packard <keithp@keithp.com>", "Patrick Lam <plam@mit.edu>"]
 license = "MIT"
 description = "Font configuration and customization library"

--- a/makefile.cargo
+++ b/makefile.cargo
@@ -90,7 +90,7 @@ $(OUT_DIR)/Makefile:
 		--disable-docs \
 		--disable-shared \
 		$(EXPAT_FLAGS) \
-		$(strip $(CONFIGURE_FLAGS)) &&
+		$(strip $(CONFIGURE_FLAGS)) && \
 		touch $(SRC_DIR)/src/fcobjshash.h
 
 else

